### ISSUE TITLE
Using actual value of conditions in text

### ISSUE THREAD

**Issue by FASTNAVi** [NPBruce/valkyrie#1017]
Hi all,

I have a question. It can be possible to show in some text window actual number from the condition?

Example:

It will be 50 enemies in one scenario. On start of the game I will set "numberofenemies=50". Every death of enemy will cause "numberofenemies=-1". On start of the turn will be text like this "It is still XX enemies comming." where XX is actual value of "numberofenemies".

Another example:

It will be 12 minutes to the end of the scenario, where 1 minute is 1 round. On start I will set "countdown=12" and on the start of every turn will be "countdown=-1". Then will be text window with "You have only XX minutes." where XX is actual value of "countdown".

It something like this possible, or it is possible to make it? Or at least show empty window with this number? So it will be text window and after clicking on continue will show only this value..

I have in my mind 2 uses for that and both of them contains over hundred of this values, so it will be difficult to do that only by normal events and condition. I know, it is possible, but not elegant.

**Comment by redwolf2** [comment]
Yes, use {var:<Var>} in your event text to show the var value. See also: https://github.com/NPBruce/valkyrie/wiki/QuestIniEvent

I would suggest to create the variable, because the default is 0 and then check if it is >= 50. This should be a litte bit easier. Don't count down, but count up and check if the limit is reached.
